monitor/alerts.py

```diff
@@ -30,8 +30,15 @@ def send(webhook: str, data: dict, content_types: str = "application/json"):
     def to_slack_message(self) -> dict:
         pass
 
-    def send_to_slack(self, webhook: str):
-        data = self.to_slack_message()
+    def to_slack_workflows_message(self) -> dict:
+        pass
+
+    def send_to_slack(self, webhook: str, is_slack_workflow = False):
+        data = {}
```

Comment:
small cosmetics - no need for this declaration (the next if - else will override it anyways)